src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue

```diff
@@ -10,15 +10,77 @@
+                  You may instead choose to define your requirement by stating it in terms of an
+                  objective, use-case, or problem statement for which the CSPs will propose a
+                  solution; do this by selecting <strong>“Architectural Design Solution”</strong>
+                  below.
+                </p>
+                <p class="mt-2 mb-0">
+                  For those more versed in cloud technology or have known specific
+                  requirements, you may define your needs by identifying the specific cloud
+                  resources and/or tools that you require by choosing <strong>“Anything as a
+                  Service (XaaS)”</strong> below.
+                </p>
+                <p class="mt-2 mb-0">
+                  In addition to the above, you have the ability to add <strong>Cloud Support
+                  Packages</strong> to any of the performance areas selected. This is optional and
+                  ancillary to your selected performance areas.
+                </p>
+              </template>
+              <template
+                  v-slot:content
+                  v-else
+              >
+                <p class="mt-2 mb-0">
+                  JWCC provides flexibility in the way you define your requirements. You may
+                  choose to define your requirement by stating it in terms of an objective,
+                  use-case, or  problem statement for which the CSPs will propose a solution;
+                  do this by selecting <strong>“Architectural Design Solution”</strong> below.
+                </p>
+                <p class="mt-2 mb-0">
+                  For those more versed in cloud technology or have known specific requirements,
+                  you may define your needs by identifying the specific cloud resources and/or
+                  tools that you require by choosing <strong>“Anything as a Service (XaaS) ”
+                </strong> below.
+                </p>
+                <p class="mt-2 mb-0">
+                  In addition to the above, you have the ability to add <strong>Cloud Support
+                  Packages</strong> to any of the performance areas selected. This is optional and
+                  ancillary to your selected performance areas.
+                </p>
+              </template>
+            </ATATExpandableLink>
+          </template>
+        </ATATAlert>
         <div class="copy-max-width">
-          <p class="mb-8"
-           v-if="totalSectionsComplete === 0">
-            Through JWCC, you have the ability to set objective-based requirements, and/or 
-            you can procure specific cloud resources, tools, and support services. We’ll walk 
-            you through each performance area below to gather details for your Description of
-            Work. You’ll have an opportunity to opt out of any areas that don’t apply to your
-            acquisition.
-          </p>
           <p class="mb-8"
```

Comment:
The alert with the expandable link needs to go below this `<p>` - i.e., this `<p>` needs to stay directly under the page header.
![image](https://github.com/dod-ccpo/atat-web-ui/assets/90333349/55cfee7a-9cfc-4e77-a447-cd7dde4df406)